### PR TITLE
Added WashPo sample articles

### PR DESCRIPTION
**Pull request by andreban**
Added sample articles courtesy of Washington Post

### CODE REVIEW

File: controllers/article.js [hunk 1]
```diff
@@ -64,6 +64,17 @@ router.get('/((\\d+))', function(req, res) {
   });
 });
 
+router.get('/washingtonpost/((\\d+))', function(req, res) {
+  console.log('ble');
```
Comment (sebastianbenz) [inline controllers/article.js]: can we remove this?

Comment (andreban) [inline controllers/article.js]: Removed

**Comment by sebastianbenz** [comment]
Works well!

Some UI nits:

- move the disclaimer above the header / below the footer 
- give the disclaimer a light grey background
- "You've reached your limit ..." box has different margin than the other boxes + text
- make sure that there is space between  "You've reached your limit ..." box and the header

I think it would be nice to have a wapo specific landing page with 

- branded header and footer
- list of real articles
- wapo related routes in it's own controller. 

What do you think?

**Comment by andreban** [comment]
- Moved disclaimers and changed colours
- Tweaked margins

**Pull request merged**